wicker/testing/storage.py

```diff
@@ -41,19 +41,19 @@ def fetch_file(self, input_path: str, local_prefix: str, timeout_seconds: int =
             shutil.copy2(self._get_local_path(input_path), dest_path)
         return dest_path
 
-    def put_object_s3(self, object_bytes: bytes, s3_path: str) -> None:
-        full_tmp_path = self._get_local_path(s3_path)
+    def put_object(self, object_bytes: bytes, target_path: str) -> None:
+        full_tmp_path = self._get_local_path(target_path)
         os.makedirs(os.path.dirname(full_tmp_path), exist_ok=True)
         with open(full_tmp_path, "wb") as f:
             f.write(object_bytes)
 
-    def put_file_s3(self, local_path: str, s3_path: str) -> None:
-        full_tmp_path = self._get_local_path(s3_path)
+    def put_file(self, local_path: str, target_path: str) -> None:
+        full_tmp_path = self._get_local_path(target_path)
         os.makedirs(os.path.dirname(full_tmp_path), exist_ok=True)
         shutil.copy2(local_path, full_tmp_path)
 
 
```

Comment:
Please see my comment at the very top of this PR about why we are changing the name of the `LocalDataStorage` class used for mocking S3 storage to a more descriptive name like `TestS3LocalDataStorage`.